modules/op-using-pipelines-as-code-with-a-github-app.adoc

```diff
@@ -1,105 +1,17 @@
 // This module is included in the following assemblies:
 // * pac/using-pipelines-as-code-repos.adoc
 
-:_mod-docs-content-type: PROCEDURE
+:_mod-docs-content-type: CONCEPT
 [id="using-pipelines-as-code-with-a-github-app_{context}"]
-= Using {pac} with a GitHub App 
+= Using {pac} with a GitHub App
 
 [role="_abstract"]
 GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to {pipelines-shortname}. Cluster administrators can configure a single GitHub App for all cluster users. For GitHub Apps to work with {pac}, ensure that the webhook of the GitHub App points to the {pac} event listener route (or ingress endpoint) that listens for GitHub events.
```

Comment:
```suggestion
GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to {pipelines-shortname}. Cluster administrators can configure a single GitHub App for all cluster users. For GitHub Apps to work with {pac}, ensure that the webhook of the GitHub App points to the {pac} controller route (or ingress endpoint) that listens for GitHub events.
```
we no more call event listener can we rephrase above sentence by removing event listener ?